app/views/layouts/mail/school.html.erb

```diff
@@ -2,48 +2,18 @@
 
 <html lang="en">
   <head>
-    <%= stylesheet_link_tag 'mailer' %>
```

Comment:
The stylesheet and mailer specific assets should be cleaned up as part of this PR.